### ISSUE TITLE
DSDL plugin supports only YANG version 1

### ISSUE THREAD

**Issue by kannanwst** [mbj4668/pyang#901]
When I tried to create the DSDL schema, I was trying to follow the Yang Version 1.1 but I got the following error:

**DSDL plugin supports only YANG version 1**

I am referring the below link  https://github.com/mbj4668/pyang/issues/426
some  limitations are  mentioned  in  this ticket  #426
**--dsdl-lax-yang-version
                    Try to translate modules with unsupported YANG
                    versions (use at own risk)
This will work for many 1.1 modules, but there are some constructs that it will not handle properly.**

**Query**
==========
**Query#1** - Please confirm  whether the above limitation is  still valid or not 
**Query#2** - if the answer is yes, then this Query#2 is applicable.
                  Could you please share the list of "constructs" which is not handled when "--dsdl-lax-yang-version"  is used?

**Comment by kannanwst** [comment]
Hi, 

      could you please respond to the above queries?

Thanks 
kannan R

**Comment by llhotka** [comment]
Yes, the limitation still applies. As far as I remember, there are two types of incompatibilities:
* new XPath functions introduced in YANG 1.1 aren't supported
* leafref appearing in a grouping may not work correctly
